.github/workflows/release.yml

```diff
@@ -249,3 +249,53 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG: ${{ steps.create-release.outputs.computed-prefix }}${{ steps.create-release.outputs.version }}
+
+  check-plugin-version:
+    name: Check if plugin version exist in the registry and upload if not
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check version
+        id: check-version
+        run: |
+          set -eo pipefail
+
+          snforge_scarb_plugin_version=$(grep version crates/snforge-scarb-plugin/Scarb.toml | cut -d '"' -f 2)
+          snforge_scarb_plugin=$(curl -s https://scarbs.xyz/api/v1/index/sn/fo/snforge_scarb_plugin.json | jq --arg version $snforge_scarb_plugin_version '[.[] | select(.v == $version)] | length > 0')
```

Comment:
Nit: 
`snforge_scarb_plugin_exists`, `snforge_scarb_plugin_uploaded` or `snforge_scarb_plugin_published` would be most accurate since we're dealing with true or false statement